omod-1.9/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_9/SystemSettingResource1_9.java

```diff
@@ -109,12 +104,7 @@ public DelegatingResourceDescription getCreatableProperties() {
     public DelegatingResourceDescription getUpdatableProperties() throws ResourceDoesNotSupportOperationException {
         DelegatingResourceDescription description = super.getUpdatableProperties();
         description.removeProperty("property");
-        description.addRequiredProperty("description");
-        description.addProperty("datatypeClassname");
-        description.addProperty("datatypeConfig");
-        description.addProperty("preferredHandlerClassname");
-        description.addProperty("handlerConfig");
-        description.addRequiredProperty("value");
+        addPropertiesCommonForCreateAndUpdate(description);

```

Comment:
What about if instead of pulling this out into a method like this we just have this method do:

    DelegatingResourceDescription description = getCreatableProperties();
    description.removeProperty("property")